### PR TITLE
fix build not done

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5323

### CODE REVIEW

File: .gitlab/ci/global.yml [hunk 1]
```diff
@@ -185,6 +185,8 @@
   extends:
     - .default-cache
     - .default-before-script
+  after_script:
+    - echo "done" > ${PIPELINE_JOBS_FOLDER}/${CI_JOB_NAME}.txt
```
Comment (tkatzir) [inline .gitlab/ci/global.yml:189]: If you only need "`done`" in that file,
you could accomplish the same with just `touch`ing in file, and check that it exists.

File: Tests/scripts/check_jobs_done.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+from argparse import ArgumentParser
+from os import getenv
+import logging
+
+
+NIGHTLY_JOBS = [
```
Comment (tkatzir) [inline Tests/scripts/check_jobs_done.py:7]: Can all these be in config files?
That way, you won't need to touch a .py file.

**Review by tkatzir — COMMENTED** [review]
Looks good!
Some comments.

**Review by tkatzir — APPROVED** [review]

File: Tests/scripts/check_jobs_done.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    base_path = Path(args.job_done_files)
+    should_fail = False
+    for job in get_build_jobs():
+        job_file = base_path / f'{job}.txt'
+        logging.info(f'checking job {job} with file {job_file} in {job_file.absolute()}')
+        if not job_file.exists():
+            logging.error(f"job {job} is not done yet")
+            should_fail = True
+        elif job_file.read_text() != 'done':
+            logging.error(f"somthing isn't OK with jom name {job}")
```
Comment (daryakoval) [inline Tests/scripts/check_jobs_done.py:88]:
```suggestion
            logging.error(f"somthing isn't OK with job name {job}")
```

Comment (tkatzir) [reply Tests/scripts/check_jobs_done.py:88]:
```suggestion
            logging.error(f"something isn't OK with job name {job}")
```

**Pull request merged**